pyro/distributions/testing/naive_dirichlet.py

```diff
@@ -14,9 +14,9 @@ class NaiveDirichlet(Dirichlet):
     This naive implementation has stochastic reparameterized gradients, which
     have higher variance than PyTorch's ``Dirichlet`` implementation.
     """
-    def __init__(self, concentration):
+    def __init__(self, concentration, validate_args=False):
```

Comment:
I needed to add `validate_args` since the `.expand` in the parent method ends up could end up calling this constructor with this kwarg via `type(self)(*args, validate_params=..)`

Reply:
Makes sense, will change.